src/language/providers/closeElement.ts

```diff
@@ -27,98 +32,207 @@ export function getCloseElementProvider() {
         position: vscode.Position
       ) {
         var backpos = position.with(position.line, position.character - 1)
+        var backpos3 = position.with(position.line, position.character - 3)
```

Comment:
We should be using either `let` for mutable variables or `const` for immutable variables.